src/angularComponent/ngcOmniboxController.js

```diff
@@ -184,11 +184,11 @@ export default class NgcOmniboxController {
   }
 
   /**
-   * Select a particular suggestion item.
+   * Adds a particular suggestion item to the list of selected items.
    *
    * @param {Object} item
    */
-  selectItem(item) {
+  addToSelections(item) {
```

Comment:
Does this method name make sense if we're doing single-select? What was wrong with the original name?